### PR TITLE
Update free-programming-cheatsheets.md

### PR DESCRIPTION
**Pull request by Karanbhateja**
Added Begineers Python Cheatsheet from hakin9.org

## What does this PR do?
Add resource(s) | Improve repo

Added a new cheatsheet of python for begineers.

### Why is this valuable ?
This cheatsheet includes every part of python for begineers with easy explanation and examples.

### How do we know it's really free?
It is available on Hakin9.org - https://hakin9.org/python-cheat-sheet-for-hackers-and-developers/

### For book lists, is it a book? 
It is a cheatsheet.
[beginners_python_cheat_sheet_pcc_all.pdf](https://github.com/EbookFoundation/free-programming-books/files/9699338/beginners_python_cheat_sheet_pcc_all.pdf)

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: more/free-programming-cheatsheets.md [hunk 1]
```diff
@@ -255,6 +255,7 @@
 * [Python Regex Cheatsheet](https://www.debuggex.com/cheatsheet/regex/python) - Debuggex (HTML)
 * [Python WorldWide Cheatsheet](https://cheatography.com/davechild/cheat-sheets/python/) - Cheatography (Python)
 * [Scapy Cheat Sheet](https://www.templateroller.com/template/160817/scapy-cheat-sheet-jeremy-stretch.html) - Jeremy Stretch (PDF)
+* [Python Begineers Cheatsheet](https://hakin9.org/wp-content/uploads/2020/02/beginnersPythonCheatSheet.pdf) - No Starch (PDF)
```
Comment (eshellman) [inline more/free-programming-cheatsheets.md:258]: is there no link from the author/publisher?

Comment (Karanbhateja) [reply more/free-programming-cheatsheets.md:258]: Nope, i tried to find on where it was originally uploaded but got nothing except hacki9.org.

Comment (davorpa) [reply more/free-programming-cheatsheets.md:258]: Eric Mattes is the right author.

image

**Comment by davorpa** [comment]
Duplicated

https://github.com/EbookFoundation/free-programming-books/blob/8e5165f77867c297b67eb3501c51bafc2b0fcce6/more/free-programming-cheatsheets.md?plain=1#L252-L253

**Pull request closed without merging**